docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,124 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`DSS CLI is installed <install_DSS_CLI>` and :ref:`initialised <initialise_DSS>`.
+* Your machine has an Intel GPU.
+
+Install the Intel GPU plugin
+----------------------------
+
+To ensure DSS can utilise Intel GPUs:
+1. The Intel GPU drivers must be installed.
+2. MicroK8s must be set up to utilise the Intel GPU drivers.   
+
+Verify the Intel GPU drivers are installed in your machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To confirm that your machine has the Intel GPU drivers set up, install the `intel-gpu-tools` package:
+
+.. code-block:: bash
+
+   sudo apt install intel-gpu-tools
+
+To list the Intel GPU devices on your machine, run:
+
+.. code-block:: bash
+
+   intel_gpu_top -L
+
+If the drivers are correctly installed, you should receive information about your GPU device from the command's output.
+
+.. note::
+   For Intel discrete GPUs on older versions of Ubuntu like 22.04, you may need to perform additional steps like install a `HWE kernel <https://ubuntu.com/kernel/lifecycle>`_.   
```

Comment:
```suggestion
   For Intel discrete GPUs on older versions of Ubuntu like 22.04, you may need to perform additional steps such as installing a `HWE kernel <https://ubuntu.com/kernel/lifecycle>`_.   
```
Which version of Ubuntu are we using for this guide?

Reply:
I used 24.04, but it can be used on older versions as well (22.04 for sure). Should we specify a minimum Ubuntu version in the requirements?

Reply:
Let's go with this then: "For Intel GPUs on Ubuntu versions older than 24.04, you may need to perform additional steps such as installing a `HWE kernel <https://ubuntu.com/kernel/lifecycle>`". 
Note that I am also removing the "discrete" word. Add if it is actually important in this case. I am not aware of what implies for GPUs.

Reply:
I think it's worth keeping after my conversation with Will French, it should be only for discrete GPUs.